CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix compression support for h2c protocol ([#4944](https://github.com/opensearch-project/OpenSearch/pull/4944))
 - Support OpenSSL Provider with default Netty allocator ([#5460](https://github.com/opensearch-project/OpenSearch/pull/5460))
 - Avoid negative memory result in IndicesQueryCache stats calculation ([#6917](https://github.com/opensearch-project/OpenSearch/pull/6917))
+- Replaces ZipInputStream with ZipFile to fix ZipSlip vulnerability ([#7230](https://github.com/opensearch-project/OpenSearch/pull/7230))
```

Comment:
replace "fix ZipSlip..." by "fix CVE-..."

Reply:
I don't know of a specific CVE for this vulnerability. But I do see a group of different vulnerabilities related to zip slip exploit. Refer here for more details:
- https://github.com/snyk/zip-slip-vulnerability
- https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=directory+traversal+crafted+zip

Reply:
Ok, I don't see a CVE in the list on https://github.com/snyk/zip-slip-vulnerability either. 

I think the correct spelling is with a space though, isn't it? "Zip Slip".